File: Back/src/ProEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;
using ProEventos.Domain.Enum;
using ProEventos.Domain.Identity;

namespace ProEventos.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Descricao { get; set; }
        public DateTime? DataEvento { get; set; }
        public DateTime? DataEventoCriaco { get; set; }
        public DateTime? DataEventoAtualizacao { get; set; }
        public string Tema { get; set; }
        public string status { get; set; }       
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
       
    }
}

File: Front/ProEventos-App/src/app/models/Evento.ts
import { RedeSocial } from './RedeSocial';
import { Lote } from "./Lote";
import { Palestrante } from './Palestrante';

export interface Evento {
  id: number;
  descricao: string;
  dataEvento?: Date;
  dataEventoCriaco?: Date;
  dataEventoAtualizacao?: Date;
  tema: string; 
  status: string; 
  imagemURL: string;
  telefone: string;
  email: string;  
}
